modules/search.py

```diff
@@ -29,3 +29,20 @@ def search(api_url, file_extension, local_boolean, local_dir, search_pattern):
     r = re.compile(search_pattern)
     result = list(filter(r.match, files))
     return result
+
+def searchType(os_platform, api_url, base_url, branch, file_extension, local_boolean, local_dir, package_type):
```

Comment:
I dont understand what's the difference between the search functionality

Reply:
Search type returns all packages of a type ex. 'ide' will return Atom, VS Code, and any others that we add with that type. See the package definitions for how types are written there. I need to update the web documentation once this is merged.